PRO-7102: Merge Eligibility Criteria

KleoG opened the pull request:
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-7102


### Change description ###
This will move the eligibility page to the start of the entire application process.


**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

Commit:
PRO-7102: Merge Eligibility Criteria

This will move the eligibility page to the start of the entire application process.

jackmunkonge commented on ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json:
```diff
@@ -1,38 +1,38 @@
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorCreateApplication", "CaseFieldID": "solsSolicitorAppReference", "PageFieldDisplayOrder": 10, "DisplayContext": "MANDATORY", "PageID": "solicitorCreateApplicationPage1", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorCreateApplication", "CaseFieldID": "solsSolicitorEmail", "PageFieldDisplayOrder": 11, "DisplayContext": "MANDATORY", "PageID": "solicitorCreateApplicationPage1", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorCreateApplication", "CaseFieldID": "solsSolicitorPhoneNumber", "PageFieldDisplayOrder": 12, "DisplayContext": "OPTIONAL", "PageID": "solicitorCreateApplicationPage1", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateApplication", "CaseFieldID": "solsStartPage", "PageFieldDisplayOrder": 1, "DisplayContext": "READONLY", "PageID": "solicitorUpdateApplicationPage1", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "n"},
```
Caseeventid is wrong and so is pageid

KleoG replied:
This has been updated

Commit:
PRO-7102: Update to CaseEventID and PageID 

Update to CaseEventID and PageID from solicitorUpdateApplication to solicitorCreateApplication
